src/installer/corehost/cli/lib.cmake

```diff
@@ -5,13 +5,17 @@
 project(${DOTNET_PROJECT_NAME})
 
 include(${CMAKE_CURRENT_LIST_DIR}/common.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/hostmisc/hostmisc.cmake)
 
 add_definitions(-D_NO_ASYNCRTIMP)
 add_definitions(-D_NO_PPLXIMP)
 add_definitions(-DEXPORT_SHARED_API=1)
 
+list(APPEND SOURCES
+    ${HOSTMISC_SRC}
+)
```

Comment:
Is `libnethost` just relying on the fact that its cmake files includes `lib.cmake` before `lib_static.cmake`, so when the static library is added `SOURCES` is already updated?

Reply:
Yep. I thought I tried that, but it does seem to work. Thanks.